rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.cpp

```diff
@@ -54,7 +56,11 @@ SqliteWrapper::SqliteWrapper()
 
 SqliteWrapper::~SqliteWrapper()
 {
-  sqlite3_close(db_ptr);
+  int rc = sqlite3_close(db_ptr);
+  if (rc != SQLITE_OK) {
+    ROSBAG2_STORAGE_DEFAULT_PLUGINS_LOG_ERROR_STREAM(
+      "Could not close open database. Error code: " + std::to_string(rc));
```

Comment:
```suggestion
      "Could not close open database. Error code: " << rc);
```

Please avoid logging rc directly, can we convert this return code into a meaningful string?